doc/Programming-guide/Your-first-Kaa-application/index.md

```diff
@@ -422,14 +422,153 @@ int main(void)
 </div>
 
 <div id="app-java" class="tab-pane fade" markdown="1" >
+```java
+
+import org.kaaproject.kaa.client.DesktopKaaPlatformContext;
+import org.kaaproject.kaa.client.Kaa;
+import org.kaaproject.kaa.client.KaaClient;
+import org.kaaproject.kaa.client.SimpleKaaClientStateListener;
+import org.kaaproject.kaa.client.configuration.base.ConfigurationListener;
+import org.kaaproject.kaa.client.configuration.base.SimpleConfigurationStorage;
+import org.kaaproject.kaa.client.logging.strategies.RecordCountLogUploadStrategy;
+import org.kaaproject.kaa.schema.sample.Configuration;
+import org.kaaproject.kaa.schema.sample.DataCollection;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.io.IOException;
+import java.util.Random;
+import java.util.Timer;
+import java.util.TimerTask;
+
+/**
+ * Class implement functionality for First Kaa application
+ *
+ * @author Maksym Liashenko
+ */
+public class FirstDemo {
+
+    private static final long DEFAULT_SAMPLE_TIME = 1000L;
+
+    private static final Logger LOG = LoggerFactory.getLogger(FirstDemo.class);
+    private static Timer timer = new Timer();
+    private static KaaClient kaaClient;
```

Comment:
If it not static, we can use it only via Singleton. This task don't need Singleton.
Another cases:
1) field in main method
uncompile, because SimpleKaaClientStateListener need kaaClient. No methods to set listener after creating kaaClient 
kaaClient = Kaa.newClient(desktopKaaPlatformContext, new SimpleKaaClientStateListener(), true);
2) field in class without static
Must be created via FirstKaaDemo class, because of main is static and we haven't object

I can't see another case. Am I wrong?